includes/Modules/Sign_In_With_Google.php

```diff
@@ -45,6 +46,18 @@ final class Sign_In_With_Google extends Module implements Module_With_Assets, Mo
 	 */
 	const MODULE_SLUG = 'sign-in-with-google';
 
+	/**
+	 * Option name to store user ID received from Google.
+	 */
+	const GOOGLE_USER_ID_OPTION = 'googlesitekitpersistent_siwg_google_user_id';
+
+	/**
+	 * Error codes.
+	 */
+	const INVALID_REQUEST_ERROR    = 'google_auth_invalid_request';
+	const INVALID_CSRF_TOKEN_ERROR = 'google_auth_invalid_g_csrf_token';
+	const SIGNIN_FAILED_ERROR      = 'google_auth_failed';
```

Comment:
Nit: I'd prefer if `ERROR` was the prefix rather than suffix.

Reply:
Updated.